framework/include/systems/NonlinearSystemBase.h

```diff
@@ -643,6 +643,9 @@ class NonlinearSystemBase : public SystemBase,
   unsigned int _current_nl_its;
   bool _compute_initial_residual_before_preset_bcs;
 
+  /// True if printing out additional information
+  bool _verbose;
```

Comment:
public member? Are you sure? What about @YaqiWang? 😄

Reply:
It is public because it gets set by `FEProblemSolve`. I guess I could get around by doing something like
```
_nl.setVerboseFlag(getParam<bool>("verbose"));
```
instead of
```
_nl._verbose = getParam<bool>("verbose");
```
Again, I was just following the existing design, but maybe I should do my own thinking sometime 😛

Reply:
In general, feel free to "fix" bad design. Make things more encapsulated when possible (more private). Avoid public members and things of that nature, even if it means going against the current design. We need to keep moving MOOSE forward.

Thanks!